src/main/resources/mcmod.info

```diff
@@ -3,12 +3,12 @@
         "modid":"Eln",
         "version": "${version}",
         "mcversion": "${mcversion}",
-        "name":"Electrical Age",
-        "url":"http://electrical-age.net/",
+        "name":"Electrical Age - jrddunbr edition",
```

Comment:
I can agree with almost everything else you've changed, but this seems a little excessive--I think leaving the name would be fine :)
```suggestion
        "name":"Electrical Age",
```

Reply:
Maybe `jrddunbr's-build`?